tests/modes.spec.ts

```diff
@@ -112,21 +112,21 @@ test("creating a new route opens a form, and auto-fill sets its name", async ()
 test("editing geometry of a polygon works", async () => {
   // Create a polygon
   await page.getByRole("button", { name: "New polygon (snapped)" }).click();
-  await clickMap(page, 241, 579);
+  await clickMap(page, 241, 509);
```

Comment:
The basemap controls bottom-left got larger, so we can't click so far down the map anymore with Playwright's default browser height!